src/schedule/zephyr_dma_domain.c

```diff
@@ -309,12 +334,16 @@ static int zephyr_dma_domain_register(struct ll_schedule_domain *domain,
 
 	if (ret < 0) {
 		tr_err(&ll_tr,
-		       "failed to register DMA IRQ for pipe task %p on core %d",
-		       pipe_task, core);
+		       "failed to register DMA IRQ for pipe task %p on core %d reg: %d",
+		       pipe_task, core, pipe_task->registrable);
 
 		return ret;
 	}
 
+	/* don't let non-registrable tasks go further */
```

Comment:
this comment doesn't add any information - it simply restates the next statement. If a comment is needed here, it should explain something that isn't obvious from the code

Reply:
You're right. Will remove or rework it.